src/Components/Blazor/Blazor/src/Http/WebAssemblyHttpMessageHandlerOptions.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+
+namespace Microsoft.AspNetCore.Blazor.Http
+{
+    /// <summary>
+    /// Configures options for the WebAssembly HTTP message handler.
+    /// </summary>
+    public static class WebAssemblyHttpMessageHandlerOptions
+    {
+        /// <summary>
+        /// Gets or sets the default value of the 'credentials' option on outbound HTTP requests.
+        /// Defaults to <see cref="FetchCredentialsOption.SameOrigin"/>.
+        /// </summary>
+        public static FetchCredentialsOption DefaultCredentials
```

Comment:
I know this is not going to survive API review eventually, but it's the closest thing to what we already have. I don't want to design a new thing just as part of reacting to an external change.